extension/packages/nextjs/app/eip-712/page.tsx

```diff
@@ -0,0 +1,164 @@
+"use client";
+
+import { useEffect, useState } from "react";
+import Link from "next/link";
+import type { NextPage } from "next";
+import useSWRMutation from "swr/mutation";
```

Comment:
Remove this ?

Reply:
oops [f4aeba5](https://github.com/scaffold-eth/create-eth-extensions/pull/3/commits/f4aeba50000c1e4f6722fe3c9beb3f15bc54f777)